fastlane/spec/private_public_fastfile_spec.rb

```diff
@@ -25,6 +25,7 @@
       end
 
       it "doesn't expose the private lanes in `fastlane lanes`" do
+        require 'fastlane/lane_list'
```

Comment:
Yes 👍 I mean what is the benefit in this case? Why do you think it's better? I'm just curious.